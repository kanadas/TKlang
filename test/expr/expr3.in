(\x y -> x & y) !(true | false & true) true
