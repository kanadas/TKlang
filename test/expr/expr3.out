expr/expr3.in
False
